decision-records/2024-12-03--creating-issues-in-github-and-jira.md

```diff
@@ -0,0 +1,53 @@
+# Creating issues in GitHub and Jira
+
+**Date:** 2024-12-03<br/>
+**Authors:** @simskij
+
+## Context and Problem Statement
+
+As the team have grown, we've ended up diverging in our practices on how to create issues in our
+repositories. To be able to work efficiently and consistently with our issues, we need to follow some
+common practices, so that the workflow remains the same across issues.
+
+## Decision
+
+These rules only apply to tickets opened by us. If someone else opens the ticket, we should
+rephrase it as part of the triage.
+
+### Issue Titles
+
+- Issues are always to be created in GitHub, unless confidentiality requirements dictate something
+  else. However, this is almost never the case. If you are unsure - ask @simskij or @sed-i.
+- Issues of the story/enhancement type should have a title that describes the work to be done in
+  the _imperative modus_, i.e "Add a bip to the bop", rather than "We should add a bip to the bop"
+- Issues of the bug type should have a title that describes the bug occuring, written in
+  this format: "When <condition> then <outcome>".
```

Comment:
It's surpisingly hard to describe complex scenarios that way. Just started writing a bug named 

"When charm is related to grafana-agent without a connected tracing backend then cos-agent exposes an inaccessible tracing URL" which doesn't sound easy to say in a status update. I think it makes sense as a guideline, but not necessarily a rule.